crates/holochain/src/conductor/interface/websocket.rs

```diff
@@ -479,7 +482,12 @@ pub mod test {
                     assert_eq!(
                         info,
                         vec![NetworkInfo {
-                            fetch_pool_info: FetchPoolInfo::default()
+                            fetch_pool_info: FetchPoolInfo::default(),
+                            number_of_peers: 1,
+                            arc_size: 1.0,
+                            total_peers: 1,
+                            bytes_since_last_time_queried: 1844,
```

Comment:
Instead of hardcoding this, I could assert only the `fetch_pool_info` as before or assert `bytes_since_last_time_queried > 0`.